README.md

```diff
@@ -60,6 +60,25 @@ See the [issue tracker] if you're unsure where to start, or feel free to reach o
 
 See also: the Kubernetes [community page].
 
+## Why kind?
+
+ - kind supports building official Kuberbetes release builds from source
+ - kind ships upstream Kubernetes release builds
```

Comment:
now that I think about it, this particular point might also be misleading, the builds in our "official" kind images ~~ match how upstream builds but are actually happening on my devices currently. let's add it back when that changes. I think the above point covers it.